libraries/annotationprocessorconsumerkotlin/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: "kotlin"
+
+dependencies {
+    annotationProcessor project(":libraries:annotationprocessor")
+    annotationProcessor deps.apt.autoValue
+    annotationProcessor project(":libraries:annotationprocessoroverexistingap")
```

Comment:
replace `annotationProcessor` usages with `kapt` and I am able to see the plugin argument generated. the build was successful as well.